ssl/ssl_sess.c

```diff
@@ -20,6 +20,7 @@
 #include "internal/ssl_unwrap.h"
 #include "ssl_local.h"
 #include "statem/statem_local.h"
+#include "internal/quic_types.h"
```

Comment:
may be we can include the file conditionally like adding a `ifndef OPENSSL_NO_QUIC`